app/containers/Forms/FormArrayFieldTemplate.tsx

```diff
@@ -13,8 +13,12 @@ const FormArrayFieldTemplate: React.FunctionComponent<ArrayFieldTemplateProps> =
   return (
     <div className={className}>
       {items.map(element => {
+        const zeroEmission =
+          element.children.props.formData.annualEmission === 0
+            ? 'zero-emission'
+            : '';
```

Comment:
This ArrayFieldTemplate is a generic element and should not have this. Maybe this should be in a `GasField` or `EmissionSourceField`? The latter would allow you to hide the emission source type name when all of its gases are at 0